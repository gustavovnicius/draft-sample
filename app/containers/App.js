import React from 'react'
import { Button } from 'rebass'

function App() {
  return (
    <div>
      <Button
        backgroundColor="primary"
        color="white"
        inverted
        rounded
      >
        Click me!
      </Button>
    </div>
  )
}

export default App;
