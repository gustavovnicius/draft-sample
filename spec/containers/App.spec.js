import React from 'react';
import renderer from 'react-test-renderer';

import App from 'containers/App.js';

describe('App', () => {
  it('renders', () => {
    const app = renderer.create(<App />);
    let tree = app.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
