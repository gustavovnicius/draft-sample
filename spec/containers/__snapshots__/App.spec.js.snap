exports[`App renders 1`] = `
<div>
  <button
    className="Button"
    href={undefined}
    style={
      Object {
        "backgroundColor": "#08e",
        "border": 0,
        "borderRadius": 2,
        "boxSizing": "border-box",
        "color": "#fff",
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "inherit",
        "fontSize": 14,
        "fontWeight": 600,
        "lineHeight": "16px",
        "margin": 0,
        "minHeight": 32,
        "paddingBottom": 8,
        "paddingLeft": 16,
        "paddingRight": 16,
        "paddingTop": 8,
        "textDecoration": "none",
      }
    }>
    Click me!
  </button>
</div>
`;
